torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -2084,6 +2084,27 @@ RegisterOperators reg2({
           return 0;
         }),
 
+    Operator(
+        "aten::divmod(int x, int y) -> (int, int)",
+        [](Stack& stack) {
+          int64_t a, b;
+          lldiv_t divresult = {};
+          pop(stack, a, b);
+          divresult = lldiv(a, b);
+          push(stack, static_cast<int64_t>(divresult.quot), \
+            static_cast<int64_t>(divresult.rem));
+          return 0;
+        }),
+      Operator(
+        "aten::divmod(float x, float y) -> (float, float)",
+        [](Stack& stack) {
+          double a, b;
+          pop(stack, a, b);
+          double rem = fmod(a, b);
+          push(stack, (a - rem)/b, rem);
+          return 0;
+        }),
+
```

Comment:
what about mixed operand types? like (int, float) and (float, int)? Can you also add these two? we should apply binary arithmetic operators rules for those use cases.

Reply:
`Apply binary arithmetic operators rules ` would it mean calculating quotient first using `/` and then calculating mod using it?

Reply:
@pandeykartikey  Yep that's correct. In generally results returned here should match python `divmod` results in all combinations of operand types. Given there will be similar combo of them, it's the best to be generated (similar to `aten::log`) instead of writing each `Operator` separately with similar implementation.